.github/workflows/healthchecks_azurekeyvault_cd.yml

```
@@ -8,22 +8,8 @@ on:
 
 jobs:
   build:
-    env:
+    uses: ./.github/workflows/reusable_cd_workflow.yml
```

```suggestion
    uses: ./.github/workflows/reusable_cd_workflow.yml
    secrets: inherit
```
We will need this so we can use ${{secrets.NUGET_API_KEY}}